test/spmd/tensor/test_tp_sharding_ops.py

```diff
@@ -127,7 +127,6 @@ def test_replicated_permute(self):
         new_dt = dist_tensor.permute(1, 0, 2)
         self.assertTrue(new_dt.placements[0].is_replicate())
         self.assertEqual(new_dt.to_local(), tensor.permute(1, 0, 2))
-        self.assertEqual(new_dt.stride(), tensor.permute(1, 0, 2).stride())
```

Comment:
Why do we remove this?

Reply:
sorry that was a accidental delete, added back